homeassistant/components/waqi/sensor.py

```diff
@@ -108,6 +108,7 @@ def __init__(self, client, station):
             self.station_name = None
 
         self._data = None
+        self._unique_id = f"{location_name}-{self.uid}"
```

Comment:
We shouldn't use location name. It doesn't fit the requirement of a unique ID:  https://developers.home-assistant.io/docs/en/entity_registry_index.html#unique-id-requirements

`uid` should be unique on its own, so ok to use?

Reply:
Yes it is, should've gone with that from the beginning.  Done now.